### DISCUSSION TITLE
Text and auto-mode speed

### DISCUSSION THREAD

**Discussion by Vashy-777** [dialogic-godot/dialogic#1125] [Q&A]
So I been looking through the code and all I can find was the "Definition" for theme_text_speed. I'm trying to use two different Hsliders to control the text speed and the text auto-mode speed; "Autoplay. I'm still a little new at all this but any help would be grateful. Thank you.

**Comment by coppolaemilio** [comment]
Hey, sorry I missed this message completely. Are you talking about the Dialogic game template? or just the plugin?
